### PR TITLE
feat: Adds lockers count

### PR DESCRIPTION
**Pull request by gluneau**
1 new indexes
  - uniqueLockerAddresses with amount

1 new data point added to tvlAggregatedDaily
  - lockersCount

### CODE REVIEW

File: src/mapping/tvl.ts [hunk 1]
```diff
@@ -3,42 +3,48 @@ import { Event, ProcessorContext } from "../processor";
...
   entities: Entities
 ): Promise<void> {
   const amount = BigInt(event.args.amount);
+  const address = getSs58Address(event.args.account);
   let lockAmount;
 
-  if (
-    event.name === events.dappStaking.unlocking.name // ||  event.name === events.dappStaking.claimedUnlocked.name
-  ) {
+  // ||  event.name === events.dappStaking.claimedUnlocked.name
```
Comment (bobo-k2) [inline src/mapping/tvl.ts:21]: Remove

File: src/processor.ts [hunk 2]
```diff
@@ -12,21 +12,21 @@ import { lookupArchive } from "@subsquid/archive-registry";
...
 const blockRange = { from: parseInt(process.env.BLOCK_RANGE!, 10) };
-const archive =
-  process.env.ARCHIVE != ""
-    ? lookupArchive(process.env.ARCHIVE!, {
-        type: "Substrate",
-        release: "ArrowSquid",
-      })
-    : undefined;
-console.log(`Archive: ${archive}`);
+// const archive =
```
Comment (bobo-k2) [inline src/processor.ts:15]: Also remove if not used. Do the same for all commented code

**Review by bobo-k2 — APPROVED** [review]

File: src/mapping/tvl.ts (hunk 1 above)
Comment (gluneau) [reply src/mapping/tvl.ts:21]: I will

File: src/processor.ts (hunk 2 above)
Comment (gluneau) [reply src/processor.ts:15]: This one, I want to keep.  in theory this should work. But something is broken on the archive side with subsquid.  I will address it with them, then uncomment this code.

**Pull request merged**